src/app/services/species.service.spec.ts

```diff
@@ -2,17 +2,16 @@ import { HttpClient, HttpClientModule } from '@angular/common/http';
 import { TestBed } from '@angular/core/testing';
 import { of } from 'rxjs';
 import species from '../Species.json';
-
 import { SpeciesService } from './species.service';
 
 describe('SpeciesService', () => {
   let service: SpeciesService;
-  let httpClientSpy: {get: jasmine.Spy};
+  let httpClientSpy: { get: jasmine.Spy };
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [ HttpClientModule ],
-      providers: [ HttpClient ],
+      imports: [HttpClientModule],
+      providers: [HttpClient],
```

Comment:
Not clear why you have imported `HttpClient`, since you shouldn't. need. `HttpClientModule` replaces it so that your app doesn't make live calls to the API. Here's an example of 2 ways to test services (there are 2 test suites in this file):

https://github.com/gSchool/ng-rickdiculous-app/blob/solution/src/app/services/episodes.service.spec.ts

Reply:
We've corrected this